packages/flame_riverpod/example/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: flame_riverpod_example
+description: Showcasing the flame_riverpod functionality.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+version: 1.0.0+1
+environment:
+  sdk: '>=2.18.5 <3.0.0'
+dependencies:
+  flame: ^1.10.1
+  flame_riverpod:
+    path: ../
```

Comment:
Since this is within the monorepo now it should just say the latest version here and Melos will connect it with path through bootstrap (and update it when we run `melos version`)